server/sonar-db-dao/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -92,8 +92,16 @@ public long countRootUsersButLogin(DbSession dbSession, String login) {
     return mapper(dbSession).countRootUsersButLogin(login);
   }
 
+  /**
+   * @deprecated only used in tests
+   */
+  @Deprecated
   public UserDto insert(DbSession session, UserDto dto) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)